src/test/java/step3/CarTest.java

```diff
@@ -28,7 +28,7 @@ public void getPosition() {
     @DisplayName("이동한다.")
     public void move() {
         int currentPosition = car.getPosition();
-        assertThat(car.move(new MoveOneForwardStrategy()).getPosition()).isIn(currentPosition, currentPosition + 1);
+        assertThat(car.move(new MoveOneForwardStrategy()).getPosition()).isEqualTo(currentPosition + 1);
```

Comment:
```suggestion
        @Test
    @DisplayName("이동한다.")
    public void move() {
        int currentPosition = car.getPosition();
        assertThat(car.move(() -> 1).getPosition()).isEqualTo(currentPosition + 1);
    }

    @Test
    @DisplayName("이동하지 않는다.")
    public void stop() {
        int currentPosition = car.getPosition();
        assertThat(car.move(() -> 0).getPosition()).isEqualTo(currentPosition );
    }
```
이동전략을 주입할 수 있게 되었으니 람다를 활용해서 다양한 테스트 코드를 작성할 수 있을것 같아요. 
컴파일시점이 아닌 런타임에 이동전략이 결정되게 되는 장점을 누려보세요 :)

Reply:
추가적으로 실패하는 테스트 코드가 존재하니 확인부탁드려요 :)